web_monitoring/db.py

```diff
@@ -434,18 +434,18 @@ def diff_version(version_uuid, versions, diffs, source_type,
                  source_metadata)
 
 
-class WebVersioningException(Exception):
+#class WebVersioningException(Exception):
```

Comment:
The problem is
Just commenting the '...' would leave the class empty
I commented the Empty class that's it.
It doesn't give error since hasn't been referred.
Anyway if needed, I'll uncomment that part. @danielballan